docs/conf.py

```diff
@@ -97,7 +97,7 @@
 #show_authors = False
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'sphinx'
+pygments_style = 'default'
```

Comment:
I changed this for local testing purposes and it remained here. Anyway, I can revert it back, but this one looks better on readthedocs.

Reply:
In try to use the same theme and styles on all open source projects from ESSS.
I will ask to revert this.

Reply:
Sure, changed to `sphinx`.